source/blog/2018-06-29-ember-3-2-released.md

```diff
@@ -0,0 +1,313 @@
+
+There are four (4) new features and zero (0) deprecations in the Ember Data 3.2 release.
+
+#### New Features (4)
+
+##### Lazy Relationship Payloads (1 of 4)
+
+In Ember Data 2.14 lazy-relationship parsing was introduced. Because this parsing used left-side/right-side keying this ment it was not compatible with polymorphic relationships.
+
+With Ember Data 3.2 this is [now fixed](https://github.com/emberjs/data/pull/5230).
+
+##### Ember Data Feature Flag Removal (2 of 4)
+
+The current feature flags for Ember Data has gone stale, therefore they are all [removed in Ember Data 3.2](https://github.com/emberjs/data/pull/5384). Ember Data is going to attempt to go in a different direction with some of the planned changes for 2018.
+
+If your app depends on enabling these feature flag to run, please reach out to the Ember Data team by opening a github issue on the [Ember Data repo](https://github.com/emberjs/data/issues) and the Ember Data team will try to assist with the transition.
+
+##### Feature Flag `ds-improved-ajax` (3 of 4)
+
+The Ember Data team has released an addon that will support the `ds-improved-ajax` API.
+
+##### Feature Flag `ds-pushpayload-return` (4 of 4)
+
+If you rely on the `ds-pushpayload-return` feature flag, you can use the following pattern to manually serialize the API response and push the record into the store.
+
+```javascript
+export function pushPayload(store, modelName, rawPayload) {
+   let ModelClass = store.modelFor(modelName);
+   let serializer = store.serializerFor(modelName);
+
+   let jsonApiPayload = serializer.normalizeResponse(store, ModelClass, rawPayload, null, 'query');
+
+  return store.push(jsonApiPayload);
+}
+```
+
+```diff
++import { pushPayload } from '<app-name>/utils/push-payload';
+
+...
+
+-this.get('store').pushPayload(modelName, rawPayload);
++pushPayload(this.get('store'), modelName, rawPayload);
+```
+
+#### Deprecations (0)
+
+There are no deprecations introduced in Ember Data 3.2.
+
+For more details on changes in Ember Data 3.2, please review the
+[Ember Data 3.2.0 release page](https://github.com/emberjs/data/releases/tag/v3.2.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI separately from Ember.js and Ember Data! To upgrade your projects using `yarn` run:
+
+```bash
+yarn upgrade ember-cli
+```
+
+To upgrade your projects using `npm` run:
+
+```bash
+npm install --save-dev ember-cli
+```
+
+After running the upgrade command run `ember init` inside of the project directory to apply the blueprint changes. You can preview those changes for [applications](https://github.com/ember-cli/ember-new-output/compare/v3.1.0...v3.2.0) and [addons](https://github.com/ember-cli/ember-addon-output/compare/v3.1.0...v3.2.0).
+
+### Changes in Ember CLI 3.2
+
+There are three (3) new features and one (1) deprecation in the Ember CLI 3.2 release.
```

Comment:
There is only one new feature listed below. Might be missing other new features or this number is a typo.

Reply:
Probably a typo, thank you for catching it!